tobac/tests/test_tracking.py

```diff
@@ -128,6 +133,158 @@ def test_linking_trackpy():
         expected_out_feature.sort_index(axis=1), actual_out_feature.sort_index(axis=1)
     )
 
+    # Test 3D tracking of a simple moving feature with periodic boundaries on hdim_1
+    test_feature = tobac.testing.generate_single_feature(
+        1,
+        1,
+        start_v=1,
+        min_h1=0,
+        max_h1=10,
+        min_h2=0,
+        max_h2=10,
+        frame_start=0,
+        num_frames=8,
+        spd_h1=3,
+        spd_h2=1,
+        spd_v=1,
+        PBC_flag="hdim_1",
+    )
+
+    expected_out_feature = copy.deepcopy(test_feature)
+    expected_out_feature["cell"] = 1
+
+    actual_out_feature = tobac.tracking.linking_trackpy(
+        test_feature,
+        None,
+        1,
+        1,
+        dz=1,
+        min_h1=0,
+        max_h1=10,
+        min_h2=0,
+        max_h2=10,
+        v_max=4,
+        method_linking="random",
+        vertical_coord=None,
+        PBC_flag="hdim_1",
+    )
+    # Just want to remove the time_cell column here.
+    actual_out_feature = actual_out_feature[
+        ["hdim_1", "hdim_2", "vdim", "frame", "feature", "time", "cell"]
+    ]
+    expected_out_feature = convert_cell_dtype_if_appropriate(
+        actual_out_feature, expected_out_feature
+    )
+
+    assert_frame_equal(
+        expected_out_feature.sort_index(axis=1), actual_out_feature.sort_index(axis=1)
+    )
+
+    # Test 3D tracking of a simple moving feature with periodic boundaries on hdim_2
+    test_feature = tobac.testing.generate_single_feature(
+        1,
+        1,
+        start_v=1,
+        min_h1=0,
+        max_h1=10,
+        min_h2=0,
+        max_h2=10,
+        frame_start=0,
+        num_frames=8,
+        spd_h1=1,
+        spd_h2=3,
+        spd_v=1,
+        PBC_flag="hdim_2",
+    )
+
+    expected_out_feature = copy.deepcopy(test_feature)
+    expected_out_feature["cell"] = 1
+
+    actual_out_feature = tobac.tracking.linking_trackpy(
+        test_feature,
+        None,
+        1,
+        1,
+        dz=1,
+        min_h1=0,
+        max_h1=10,
+        min_h2=0,
+        max_h2=10,
+        v_max=4,
+        method_linking="random",
+        vertical_coord=None,
+        PBC_flag="hdim_2",
+    )
+    # Just want to remove the time_cell column here.
```

Comment:
the same but maybe more generic in case the column names would be different:  `actual_out_feature = actual_out_feature.drop('time_cell', axis=1)`